pkg/envtest/helper.go

```diff
@@ -3,10 +3,21 @@ package envtest
 import (
 	"reflect"
 
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+var (
+	crdScheme = runtime.NewScheme()
+)
+
+func init() {
+	_ = apiextensionsv1.AddToScheme(crdScheme)
```

Comment:
You could use:

```go
import apiextensionsinstall "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install"

apiextensionsinstall.Install(crdScheme)
```

Reply:
Ah nice, didn't know that existed. Fixing soon!

Reply:
wow TIL this affects a lot of my own code :joy:

Reply:
Just FYI that `Install` will panic on an error, which is okay since this is envtest.